apps/datahub/src/app/record/record-metadata/record-metadata.component.ts

```diff
@@ -77,4 +83,9 @@ export class RecordMetadataComponent {
       .getFiltersForOrgs([org])
       .subscribe((filters) => this.searchService.updateFilters(filters))
   }
+
+  openLightbox(src: string) {
+    this.lightboxInstance = basicLightbox.create(`<img src="${src}"/>`)
+    this.lightboxInstance.show()
```

Comment:
Wondering if we could just do `basicLightbox.create(`<img src="${src}"/>`).show()` and get rid of the `lightboxInstance: any`?

Reply:
Yes this works :+1: